### DISCUSSION TITLE
Support for Prestashop v8 and multistore?

### DISCUSSION THREAD

**Discussion by minhcz** [vuestorefront/prestashop#92] [Q&A]
Hello,

Do you guys have plan for support Prestashop version 8 and Multistore?

Thank you

**Comment by samberrry** [comment]
Hello, 
as most of the PrestaShop modules do not support v8 and the company website offers 1.7.x versions as well, it's still not supported. About multi-store, it's interesting feature, we are looking forward to see some supporting candidates for this feature.

thanks

**Reply by cloud-construct** [reply]
We've been impressed by Vue Storefront, but have a requirement for multi-store to represent different configurations for various territories.
Is Multistore on Prestashop on the dev roadmap?
